tests/umap-sparsestore/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+
+#############################################################################
+# Copyright 2017-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
update to 2021 copyright

Reply:
I've updated the copyrights and pushed the changes.
Thanks!